src/doc/src/reference/unstable.md

```diff
@@ -1882,6 +1883,16 @@ Specify which packages participate in [feature unification](../reference/feature
 * `package` _(unimplemented)_: Dependency features are considered on a package-by-package basis,
   preferring duplicate builds of dependencies when different sets of features are activated by the packages.
 
+## package list JSON format
+
+* Tracking Issue: [#11666](https://github.com/rust-lang/cargo/issues/11666)
+
+The `--list json` flag value in `cargo package` emits machine-readable JSON 
+information about each file included in a package.
+Requires `-Zunstable-options`.
+See [`cargo package --list`](../commands/cargo-package.md#option-cargo-package---list[fmt])
+for more information.
```

Comment:
Can we document the schema?

Reply:
That is in `cargo package --list`. Do we want to duplicate it?

Reply:
Oh, forgot that we only document the schemas in the relevant commands, nm